### PR TITLE
Switched to swagger 2.0 and yaml.. feel free to delete annotations

### PR DESCRIPTION
**Pull request by jacarey**

### CODE REVIEW

**Comment by mcovarr** [comment]
You rock!  :smile:

File: src/main/resources/swagger/cromwell.yaml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        - google_oauth:
+            - openid
+securityDefinitions:
+  google_oauth:
+    type: oauth2
+    authorizationUrl: 'https://accounts.google.com/o/oauth2/auth'
+    flow: implicit
+    scopes:
+      openid: open id authorization
+definitions:
```
Comment (mcovarr) [inline src/main/resources/swagger/cromwell.yaml]: @cjllanwarne a `definitions` section like this would be nice for Thurloe

**Comment by mcovarr** [comment]
Assuming it all still works with the annotations removed (which I have not tried) :+1:

**Comment by jacarey** [comment]
I was able to hit endpoints but wasn't sure about testing since I have no idea how to get a valid workflow id.

**Comment by geoffjentry** [comment]
I'm going to do the annotations & make sure it works via local, will add to this PR when complete

**Comment by geoffjentry** [comment]
I've removed the annotations, we're now all yaml all the time

**Comment by geoffjentry** [comment]
:+1:  I think we should wait until any web service tickets in flight are merged in though

**Comment by mcovarr** [comment]
Note this PR is against master but should be against develop!

The two endpoint tickets in flight are @scottfrazer's workflow options #176 and my call metadata sketch  #182.  For mine I'd definitely prefer to have these YAML changes merged to develop first; I wouldn't expect any serious conflicts with Scott's stuff either.

**Comment by geoffjentry** [comment]
Good catch!

My preference would be to merge sooner but thought the individuals would rather not fiddle with their PRs, so whatever works best is fine by me

**Comment by geoffjentry** [comment]
@jacarey as per what @mcovarr said, can you update the PR to be against develop?

**Pull request closed without merging**